prerequisites.tf

```diff
@@ -39,16 +39,18 @@ module "iam" {
   secret_prefix                   = local.secret_prefix
   tiering_enable_obs_integration  = var.tiering_enable_obs_integration
   additional_iam_policy_statement = var.additional_instance_iam_policy_statement
+  enable_ssm_policy               = var.vpc_endpoint_ssm_create
 }
 
 module "vpc_endpoint" {
-  count                          = var.vpc_endpoint_ec2_create || var.vpc_endpoint_proxy_create || var.vpc_endpoint_s3_gateway_create ? 1 : 0
+  count                          = var.vpc_endpoint_ssm_create || var.vpc_endpoint_ec2_create || var.vpc_endpoint_proxy_create || var.vpc_endpoint_s3_gateway_create ? 1 : 0
```

Comment:
need to also add `var.vpc_endpoint_lambda_create`

Reply:
removed